### ISSUE TITLE
Improve burndown section

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#15034]
The burndown section has pretty much stayed as-is since we first implemented it, but it really just was an MVP to have _something_. I propose we should do the following:

- [ ] Make the date-range adjustable **1d**
- [ ] Better support for changes within a day (more fine-grained response) **1d**
  - If you open, approve, and merge a changeset within 24h, it will only be shown as "merged" in the graph, making it look pretty unspectacular
- [ ] Better visualization, the block diagram probably doesn't work too well (I don't know what does better, though) **1d**
- [ ] More modes of visualization
  - All states in one diagram (what we currently have)
  - All left open in a burn-down chart
  - Activity graph(?) How much change during a day, how many approved, merged etc. Higher means more activity towards getting this campaign done
  - **more awesome suggestions here**

**Comment by chrispine** [comment]
@eseliger Could you please add an estimate to this? Thanks!

**Comment by eseliger** [comment]
I added estimates to the bullet points, since I think they can be worked on individually and don't really depend on each other.

For the better visualization, I think it's not too hard to implement, once we settled on something, but in the past when doing some quick researches, I couldn't find something that looked super compelling to me.

For the "More modes of visualization" I can't tell yet, I think we should decide what we really want here first. If it's just simple graphs that are subsets of what we already have, it's probably fairly easy. If it requires some new compute methods in the backend, it might get slightly more complex.
